Tests/FoundationEssentialsTests/NotificationCenterTests.swift

```diff
@@ -0,0 +1,229 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+
+#if FOUNDATION_FRAMEWORK
+@testable import Foundation
+#else
+@testable @_spi(SwiftCorelibsFoundation) import FoundationEssentials
+#endif
+
+import Testing
+
+fileprivate final class TestObject: Sendable {}
+
+@Suite("NotificationCenter", .timeLimit(.minutes(1)))
+private struct NotificationCenterTests {
```

Comment:
I think you'll want to name this something different than the other test suite so that the tests are distinguishable